src/scripts/dashboardpage.js

```diff
@@ -1,447 +1,928 @@
+
+    window.DashboardPage = {
+        renderPaths: function (page, systemInfo) {
+            page.querySelector("#cachePath").innerHTML = systemInfo.CachePath;
+            page.querySelector("#logPath").innerHTML = systemInfo.LogPath;
+            page.querySelector("#transcodingTemporaryPath").innerHTML = systemInfo.TranscodingTempPath;
+            page.querySelector("#metadataPath").innerHTML = systemInfo.InternalMetadataPath;
+        },
+        startInterval: function (apiClient) {
+            apiClient.sendMessage("SessionsStart", "0,1500");
+            apiClient.sendMessage("ScheduledTasksInfoStart", "0,1000");
+        },
+        stopInterval: function (apiClient) {
+            apiClient.sendMessage("SessionsStop");
+            apiClient.sendMessage("ScheduledTasksInfoStop");
+        },
+        getSessionNowPlayingStreamInfo: function (session) {
+            var html = "";
+            var showTranscodingInfo = false;
+            var displayPlayMethod = playMethodHelper.getDisplayPlayMethod(session);
+
+            if ("DirectStream" === displayPlayMethod ? (html += globalize.translate("sharedcomponents#DirectStreaming"), true) : "Transcode" == displayPlayMethod ? (html += globalize.translate("sharedcomponents#Transcoding"), session.TranscodingInfo && session.TranscodingInfo.Framerate && (html += " (" + session.TranscodingInfo.Framerate + " fps)"), showTranscodingInfo = true, true) : "DirectPlay" == displayPlayMethod && (html += globalize.translate("sharedcomponents#DirectPlaying")), showTranscodingInfo) {
+                var line = [];
+
+                if (session.TranscodingInfo) {
+                    if (session.TranscodingInfo.Bitrate) {
+                        if (session.TranscodingInfo.Bitrate > 1e6) {
+                            line.push((session.TranscodingInfo.Bitrate / 1e6).toFixed(1) + " Mbps");
+                        } else {
+                            line.push(Math.floor(session.TranscodingInfo.Bitrate / 1e3) + " kbps");
+                        }
+                    }
+
+                    if (session.TranscodingInfo.Container) {
+                        line.push(session.TranscodingInfo.Container);
+                    }
+
+                    if (session.TranscodingInfo.VideoCodec) {
+                        line.push(session.TranscodingInfo.VideoCodec);
+                    }
+
+                    if (session.TranscodingInfo.AudioCodec && session.TranscodingInfo.AudioCodec != session.TranscodingInfo.Container) {
```

Comment:
the check of "audio_codec != container" makes little sense to me... explanation?

Reply:
I don't know honestly. Maybe it's related to https://github.com/jellyfin/jellyfin-web/pull/8/files#diff-8a9a85bc85d6ece089cfba53d6bb66b1R453. We're probably not interested in showing the container _and_ the audio codec if they're the same?